sosw/components/dynamo_db.py

```diff
@@ -592,7 +593,7 @@ def get_by_scan(self, attrs=None, table_name=None, strict=None, fetch_all_fields
         return result
 
 
-    def get_by_scan_generator(self, attrs=None, table_name=None, strict=None, fetch_all_fields=None):
+    def get_by_scan_generator(self, attrs=None, table_name=None, index_name=None, strict=None, fetch_all_fields=None):
```

Comment:
Better make the new param last

Reply:
As NGR said: "I think that in this specific case we can inject it in the middle to allow the same order of arguments in all the functions (both get and scan). I have checked the code of both sosw and tts_lambdas - should not affect anything."